### PR TITLE
Fix TravisCI builds

### PR DESCRIPTION
**Pull request by csaltos**
SNAPSHOT versions not available for finagle nor util so using finagle-6.25.0 and util-6.24.0 directly.

### CODE REVIEW

**Comment by mosesn** [comment]
@csaltos the reason why we have these is so that scrooge `develop` users can still compile when they're using features from `util` or `finagle` that haven't been published to maven central yet.  Maybe the right solution is to publish-local `util` and `finagle` instead?

**Comment by mosesn** [comment]
It's also a good point that it's hard to tell why the `+ suffix` bit has been added.  If you add a comment explaining about depending on `develop` versions of `util` and `finagle`, we could probably merge that in too.  Thanks for your contributions, we really appreciate it :+1:

**Comment by csaltos** [comment]
OK, the suffix trick is actually a good trick, but since neither util-6.24.0-SNAPSHOT nor finagle-6.25.0-SNAPSHOT are available Travis CI is constantly reporting errors for missing those dependencies.

I will add the comment you mention, I hope I'm able to explain it well.

**Comment by csaltos** [comment]
OK, ready, I hope it's correct.

**Comment by mosesn** [comment]
Do you want to try to fix the build here too: https://github.com/twitter/scrooge/blob/master/.travis.yml#L10 or just do the comment bit in this PR?

**Comment by csaltos** [comment]
I will try to fix the build at https://github.com/twitter/scrooge/blob/master/.travis.yml#L10 too

File: project/Build.scala [hunk 1]
```diff
@@ -14,8 +14,12 @@ object Scrooge extends Build {
   val suffix = if (branch == "master") "" else "-SNAPSHOT"
 
   val libVersion = "3.18.1" + suffix
-  val utilVersion = "6.24.0" + suffix
-  val finagleVersion = "6.25.0" + suffix
+
+  // For develop users: if you are using new features in util and finagle dependencies
+  // that are not yet published to Maven Central, you can publish util and finagle locally
+  // or you can add the "+ suffix" to get the SNAPSHOT versions accordingly
+  val utilVersion = "6.24.0"
```
Comment (mosesn) [inline project/Build.scala]: actually, could you leave it so that it still says "+ SNAPSHOT"?

Comment (mosesn) [inline project/Build.scala]: *+ suffix

Comment (csaltos) [inline project/Build.scala]: OK

Comment (csaltos) [inline project/Build.scala]: done

**Comment by csaltos** [comment]
Now scroog build is broken at Travis CI but I will try to fix it from .travis.yml

**Comment by csaltos** [comment]
OK, I think I fix the build at .travis.yml but I mix the PR with other commits, I'm sorry, I will create another PR with a clean commit

**Comment by csaltos** [comment]
OK, I move the Travis CI fix to the PR https://github.com/twitter/scrooge/pull/196

**Pull request closed without merging**